### PR TITLE
name update on README.md

### PR DESCRIPTION
**Pull request by sftwaredvlp**
I updated the name

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # soccer-stonks
-or just call it STONKSS
+or just call it RON STONKSS
```
Comment (Duzduran) [inline README.md:2]: Great idea Hasan, I like the humor 😀

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
```
Comment (Duzduran) [inline README.md:3]: Can you also add descriptions for how to setting up venv and installing requirement.txt here, as well as we can add info on running kedro & streamlit

**Review by Duzduran — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sftwaredvlp) [reply README.md:3]: Hey Ahmet! I made the changes that you requested.

File: README.md [hunk 3]
```diff
@@ -1,7 +1,38 @@
...
+
+Once the virtual environment is activated, install the required dependencies:
+```sh
+pip install -r requirements.txt
+```
+
+- To run kedro `kedro run`
+- To run streamlıt `streamlit run streamlit_app.py`
+
 ## Overview
```
Comment (Duzduran) [inline README.md]: Nice work Hasan, can you also remove these explanations that kedro automatically created

**Comment by Duzduran** [comment]
Hasan keep the description about how to run Jupyter

File: README.md (hunk 3 above)
Comment (sftwaredvlp) [reply README.md]: I did the changes.

**Comment by sftwaredvlp** [comment]
Ahmet I did all the changes, if you see more please hit me up otherwise lets merge ;D

**Review by Duzduran — APPROVED** [review]

**Pull request merged**